<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="delete_advertisement_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Advertisement</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Advertisement List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>View Count</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>369</width>
      <height>389</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_advertisement_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Advertisement</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="archive_advertisement_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Archive Advertisement</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="temp_advert_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp Advertisement</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filter_advertisements_bt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
